### DISCUSSION TITLE
If you're having problems with updating the AMR package beta version through github

### DISCUSSION THREAD

**Discussion by theanita1** [msberends/AMR#176] [General]
Was trying to update my AMR package through github and kept getting this error
`Error in utils::download.file(url, path, method = method, quiet = quiet, : download from 'https://api.github.com/repos/msberends/AMR/tarball/HEAD' failed`

looking at some similar error codes on stackoverflow, I tried this suggested code
```
options(timeout=9999999)
```

and tada! updated package

**Comment by msberends** [comment]
For future use, you can use our beta CRAN-like repo, which contains the latest GitHub pkg version, does not require direct download from GitHub, and is only available if the beta version can actually be installed. For that, run:

```r
install.packages("AMR", repos = "https://beta.amr-for-r.org")
```

**Comment by theanita1** [comment]
Hi @msberends 
I tried updating using the suggested code and received this error

![image](https://github.com/user-attachments/assets/68ae9119-0b92-4797-b80b-f898e8f1e6fd)

Is there something on my end I need to fix ?

**Reply by theanita1** [reply]
I ran my previous install code `remotes::install_github("msberends/AMR")` and I was able to update from 2.1.1.9233 to 2.1.1.9235

**Reply by msberends** [reply]
That’s very weird! What happens if you visit the website in your browser?

**Comment by theanita1** [comment]
It took me to a webpage with a directory of files and a photo of you...
I am running RStudio 4.3.1 if that helps with the diagnostics

**Reply by msberends** [reply]
Then it should just work, I don’t understand. What if you do this?

```r
install.packages("AMR", repos = "beta.amr-for-r.org")
```

if this doesn’t work, your R installation might be limited to certain repository addresses?
